test/models/category_test.rb

```diff
@@ -1,7 +1,26 @@
 require 'test_helper'
 
 class CategoryTest < ActiveSupport::TestCase
-  # test "the truth" do
-  #   assert true
-  # end
+  include CategoryItemsSetup
+
+  test "a category with all attributes is valid" do
+    category = Category.new(title: "Snowboards")
+
+    assert category.valid?
+  end
+
+  test "a category without a title is invalid" do
+    category = Category.new(title: "")
+
+    refute category.valid?
+  end
+
+  test 'a category has items' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.